app/routes/projects/index.tsx

```diff
@@ -14,22 +14,36 @@ import {
   Grid,
   IconButton,
   Paper,
-  Stack,
   useMediaQuery,
   useTheme,
+  AppBar,
+  Toolbar,
+  styled
 } from "@mui/material";
 import HighlightOffIcon from "@mui/icons-material/HighlightOff";
 import ExpandMore from "@mui/icons-material/ExpandMore";
 import FilterAltIcon from "@mui/icons-material/FilterAlt";
 import CloseIcon from "@mui/icons-material/Close";
-import { styled } from "@mui/material";
 import { SortInput } from "app/core/components/SortInput";
 import { searchProjects } from "~/models/project.server";
 import { requireProfile } from "~/session.server";
 import type { ProjectStatus } from "~/models/status.server";
 import { getProjectStatuses } from "~/models/status.server";
 import { ongoingStage, ideaStage } from "~/constants";
 import Link from "~/core/components/Link";
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
```

Comment:
is this variable being used?

Reply:
Yes, I used it to make the component responsive. It helped me change the breakpoints values to match the old behavior. In line 262 is where I use that variable.